cmd/opm/index/deprecatetruncate.go

```diff
@@ -25,7 +25,9 @@ var deprecateLong = templates.LongDesc(`
 		Produces the following update graph in quay.io/my/index:v2
 		1.4.0 -- replaces -> 1.3.0 [deprecated]
 		
-	Deprecating a bundle that removes the default channel is not allowed. Changing the default channel prior to deprecation is possible by publishing a new bundle to the index.
+	Deprecating a bundle that removes the default channel is not allowed unless the head(s) of all channels are being deprecated (the package is subsequently removed from the index). 
```

Comment:
How it can work for our automated process that requires deprecating many cases? 

c/c @gallettilance